client/components/MainPhotos.jsx

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import SinglePhoto from './SinglePhoto.jsx';
+import styled from 'styled-components';
```

Comment:
JavaScript doesn't care, but I like to import all of my npm modules first, and _then_ my own modules.